sdk/iot/Azure.Iot.Hub.Service/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Design", "CA1051:Do not declare visible instance fields", Justification = "<Pending>", Scope = "member", Target = "~F:Azure.Iot.Hub.Service.IoTHubServiceClient.Devices")]
```

Comment:
Instead of having public instance fields, you can declare them as properties with internal/ private setters.